docs/features/tokens/Tokens.stories.mdx

```diff
@@ -0,0 +1,234 @@
+        { token: "alias-warning-hover", variable: "--o-ui-b-alias-warning-hover", valueLight: "#4E4027", valueDark: "#65522F", itemRenderer: borderRenderer},
+        { token: "alias-warning-active", variable: "--o-ui-b-alias-warning-active", valueLight: "#2C2619", valueDark: "#4E4027", itemRenderer: borderRenderer},
+        { token: "alias-success", variable: "--o-ui-b-alias-success", valueLight: "#2D8739", valueDark: "#2D8739", itemRenderer: borderRenderer},
+        { token: "alias-success-hover", variable: "--o-ui-b-alias-success-hover", valueLight: "#284929", valueDark: "#2E5F30", itemRenderer: borderRenderer},
+        { token: "alias-success-active", variable: "--o-ui-b-alias-success-active", valueLight: "#1B2A1A", valueDark: "#284929", itemRenderer: borderRenderer }
+    ]}
+/>
+
+## Box Shadows
+
+<TokenTable
+    colors={[
+        { token: "alias-skim", variable: "--o-ui-bs-alias-skim", valueLight: "bs-1", valueDark: "bs-1", itemRenderer: boxShadowRenderer},
+        { token: "alias-lifted", variable: "--o-ui-bs-alias-lifted", valueLight: "bs-2", valueDark: "bs-2" , itemRenderer: boxShadowRenderer},
+        { token: "alias-raised", variable: "--o-ui-bs-alias-raised", valueLight: "bs-3", valueDark: "bs-3", itemRenderer: boxShadowRenderer},
+        { token: "alias-floating", variable: "--o-ui-bs-alias-floated" , valueLight: "bs-4", valueDark: "bs-4", itemRenderer: boxShadowRenderer}
+    ]}
+/>
+
+## Icon Colors
+
+<TokenTable
+    colors={[
+        { token: "alias-primary", variable: "--o-ui-icon-alias-primary", valueLight: "#111010", valueDark: "#FFFFFF", itemRenderer: iconRenderer },
+        { token: "alias-secondary", variable: "--o-ui-icon-alias-secondary", valueLight: "#434342", valueDark: "#F4F4F4", itemRenderer: iconRenderer },
+        { token: "alias-tertiary", variable: "--o-ui-icon-alias-tertiary", valueLight: "#777676", valueDark: "#E5E5E5", itemRenderer: iconRenderer },
+        { token: "alias-faint", variable: "--o-ui-icon-alias-faint", valueLight: "#929292", valueDark: "#C4C3C3", itemRenderer: iconRenderer },
+        { token: "alias-accent", variable: "--o-ui-icon-alias-accent", valueLight: "#5D65F6", valueDark: "#7689FD", itemRenderer: iconRenderer },
+        { token: "alias-alert", variable: "--o-ui-icon-alias-alert", valueLight: "#D93C37", valueDark: "#F88F8D", itemRenderer: iconRenderer },
+        { token: "alias-warning", variable: "--o-ui-icon-alias-warning", valueLight: "#F6BA42", valueDark: "#F6BA42", itemRenderer: iconRenderer },
+        { token: "alias-success", variable: "--o-ui-icon-alias-success", valueLight: "#2D8739", valueDark: "#17A93C", itemRenderer: iconRenderer },
+        { token: "alias-static-white", variable: "--o-ui-icon-alias-static-white", valueLight: "#FFFFFF", valueDark: "#FFFFFF", itemRenderer: iconRenderer }
+    ]}
+/>
+
+## Font Sizes
+
+> In Orbit, **1 rem unit = 16 pixels**.
```

Comment:
I would display this info at the top of the page liste Paste did because it's not only font sizes which use rem. For example, Radii also use rem values.